atef/widgets/config.py

```diff
@@ -1416,12 +1453,51 @@ class ComponentListWidget(StringListWithDialog):
     """
 
     _search_widget: Optional[HappiDeviceComponentWidget] = None
+    suggest_comparison: QSignal = QSignal(Comparison)
+    get_device_list: Optional[Callable[[], List[str]]]
+
```

Comment:
The default sort order in the component list is reverse alphabetical, which I think must be a default "sort descending column 1". Can this be reversed?

Reply:
I tried to fix this a few times and couldn't quite figure it out. If you can pinpoint it, I'd be very happy to fix it...

Reply:
I found a one-line fix. I'm going to rudely push to your branch and you can modify it if you like.

Reply:
I was thinking of the wrong one. I had trouble with this widget:
<img width="392" alt="image" src="https://user-images.githubusercontent.com/5139267/174170612-1afa2e66-f546-4e2a-a152-444b39f9fc78.png">
Your fix is good 👍

Reply:
I didn't even notice the ordering in that one!